cypress/support/util.ts

```diff
@@ -27,3 +27,19 @@ export function gqlOptions(body: any, additionalOptions?: any) {
     ...additionalOptions,
   };
 }
+
+export function expectHrefIdToBeInArray(link, ids: number[]) {
+  cy.wrap(link)
+    .invoke('attr', 'href')
+    .then((href) => {
+      const id = parseInt(href.match(/\d+/)[0]);
+      expect(ids).includes(id);
+    });
+}
+
+export function expectOnlyObjectIdsInLinks(objects, linksDataCy) {
+  const ids = objects.map(({ id }) => id);
```

Comment:
Again, it might be better to not bother with a named function and, instead, just write the object map and expect code where they're used (i.e. events-index.cy.ts and venues.cy.ts)

Since the code is pretty short it doesn't feel terrible to repeat it. Also, you can use `const eventIds` or `const venueIds` to make it even more explicit what's going on.

Reply:
Thanks for all these, those are great points. Before putting it in a bit convoluted, general function I had in fact ids named explicitly like this.